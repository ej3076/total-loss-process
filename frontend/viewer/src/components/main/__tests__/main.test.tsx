import React from 'react';
import { create } from 'react-test-renderer';

import Main from '../';

describe('<Main />', () => {
  it('should render correctly', () => {
    const tree = create(
      <Main>
        <h1>Hello world</h1>
      </Main>,
    );
    expect(tree).toMatchSnapshot();
  });
  it('should render with extra classnames', () => {
    const tree = create(
      <Main className="foobar">
        <h1>Hello world</h1>
      </Main>,
    );
    expect(tree).toMatchSnapshot();
  });
});
