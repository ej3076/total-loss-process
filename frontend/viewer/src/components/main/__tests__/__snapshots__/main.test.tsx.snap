// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> should render correctly 1`] = `
<main
  className="main"
  role="main"
>
  <h1>
    Hello world
  </h1>
</main>
`;

exports[`<Main /> should render with extra classnames 1`] = `
<main
  className="foobar main"
  role="main"
>
  <h1>
    Hello world
  </h1>
</main>
`;
